File: chatwoot/ruby/src/GetAccountAgentsExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
end

begin
    response = ChatwootClient::AgentsApi.new.get_account_agents(
        0, # account_id
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling AgentsApi#get_account_agents: #{e}"
end

File: chatwoot/python/src/GetAccountAgentsExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"userApiKey": "USER_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.AgentsApi(api_client).get_account_agents(
            account_id=0,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling AgentsApi#get_account_agents: %s\n" % e)
